upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -41,60 +42,87 @@ var _ fi.ModelBuilder = &BootstrapChannelBuilder{}
 
 // Build is responsible for adding the addons to the channel
 func (b *BootstrapChannelBuilder) Build(c *fi.ModelBuilderContext) error {
-	addons, manifests, err := b.buildManifest()
-	if err != nil {
-		return err
-	}
 
-	addonsYAML, err := utils.YamlMarshal(addons)
-	if err != nil {
-		return fmt.Errorf("error serializing addons yaml: %v", err)
-	}
-
-	name := b.cluster.ObjectMeta.Name + "-addons-bootstrap"
+	// manifestURL, err := a.GetManifestFullUrl()
```

Comment:
Any reason to leave the commented out section here?

Reply:
Nope, good catch. I cleaned that up.